packages/js/product-editor/src/blocks/notice/block.json

```diff
@@ -0,0 +1,42 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-notice",
+	"title": "Notice",
+	"category": "woocommerce",
+	"description": "Notice description",
+	"keywords": [ "products", "notice" ],
+	"textdomain": "default",
+	"attributes": {
+		"id": {
+			"type": "string"
+		},
+		"title": {
+			"type": "string"
+		},
+		"content": {
+			"type": "string"
+		},
+		"buttonText": {
+			"type": "string"
+		},
+		"type": {
+			"type": "string"
+		}
+	},
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": true,
+		"reusable": false,
+		"inserter": false,
+		"lock": false,
+		"__experimentalToolbar": false
+	},
+	"providesContext": {
+		"isInSelectedTab": "isSelected"
```

Comment:
Should remove this, I assume this was a copy/paste thing.

Reply:
Yeah, you're right. We won't need `isSelected` anymore. I removed it in the commit [f07760d](https://github.com/woocommerce/woocommerce/pull/39675/commits/f07760d15aa3483f936db124ec074bb94b8c2512)